### DISCUSSION TITLE
No Data in Kibana after Migration to 2.3.50 but stuck in Redis

### DISCUSSION THREAD

**Discussion by trb143** [Security-Onion-Solutions/securityonion#4078] [Unsupported Versions]
Taking our production box running 16.04 Security Onion and upgrading it to Ubuntu 18.4 and then Security Onion 2.3.50, No traffic is appearing in Kibana.
I did this on Thursday using the network install option.
The box is configured in standalone mode.

Looking at Grafana the Redis Queue is filling up and the number of documents in ES is not increasing very fast.
I have run so-elastic-clear as the old indexes were copied over and filled up the disk to see if this fixed the issue.  It did not, even after a reboot.
Looking in Kibana Discovery it would seem that all the so-indexes are missing.

Error: Could not locate that index-pattern-field (id: @timestamp)
    at FieldParamType.deserialize (https://trinity-ids/kibana/38015/bundles/plugin/data/data.plugin.js:1:661365)
    at https://trinity-ids/kibana/38015/bundles/plugin/data/data.plugin.js:1:415077
    at Array.forEach (<anonymous>)
    at AggConfig.setParams (https://trinity-ids/kibana/38015/bundles/plugin/data/data.plugin.js:1:414501)
    at new AggConfig (https://trinity-ids/kibana/38015/bundles/plugin/data/data.plugin.js:1:414363)
    at agg_configs_AggConfigs.createAggConfig (https://trinity-ids/kibana/38015/bundles/plugin/data/data.plugin.js:1:197161)
    at https://trinity-ids/kibana/38015/bundles/plugin/data/data.plugin.js:1:197513
    at Array.forEach (<anonymous>)
    at new agg_configs_AggConfigs (https://trinity-ids/kibana/38015/bundles/plugin/data/data.plugin.js:1:197492)
    at Object.createAggConfigs (https://trinity-ids/kibana/38015/bundles/plugin/data/data.plugin.js:8:195209)

The logstash log file gives the following errors;
[2021-05-03T15:15:13,914][INFO ][logstash.outputs.elasticsearch] retrying failed action with response code: 500 ({"type"=>"illegal_state_exception", "reason"=>"Pipeline processor configured for non-existent pipeline [common]"})
[2021-05-03T15:15:13,914][INFO ][logstash.outputs.elasticsearch] Retrying individual bulk actions that failed or were rejected by the previous bulk request. {:count=>96}

**Comment by weslambert** [comment]
> Taking our production box running 16.04 Security Onion and upgrading it to Ubuntu 18.4 and then Security Onion 2.3.50, No traffic is appearing in Kibana.
> I did this on Thursday using the network install option.
> The box is configured in standalone mode.
> 
> Looking at Grafana the Redis Queue is filling up and the number of documents in ES is not increasing very fast.
> I have run so-elastic-clear as the old indexes were copied over and filled up the disk to see if this fixed the issue. It did not, even after a reboot.
> Looking in Kibana Discovery it would seem that all the so-indexes are missing.
> 
> Error: Could not locate that index-pattern-field (id: @timestamp)
> at FieldParamType.deserialize (https://trinity-ids/kibana/38015/bundles/plugin/data/data.plugin.js:1:661365)
> at https://trinity-ids/kibana/38015/bundles/plugin/data/data.plugin.js:1:415077
> at Array.forEach ()
> at AggConfig.setParams (https://trinity-ids/kibana/38015/bundles/plugin/data/data.plugin.js:1:414501)
> at new AggConfig (https://trinity-ids/kibana/38015/bundles/plugin/data/data.plugin.js:1:414363)
> at agg_configs_AggConfigs.createAggConfig (https://trinity-ids/kibana/38015/bundles/plugin/data/data.plugin.js:1:197161)
> at https://trinity-ids/kibana/38015/bundles/plugin/data/data.plugin.js:1:197513
> at Array.forEach ()
> at new agg_configs_AggConfigs (https://trinity-ids/kibana/38015/bundles/plugin/data/data.plugin.js:1:197492)
> at Object.createAggConfigs (https://trinity-ids/kibana/38015/bundles/plugin/data/data.plugin.js:8:195209)
> 
> The logstash log file gives the following errors;
> [2021-05-03T15:15:13,914][INFO ][logstash.outputs.elasticsearch] retrying failed action with response code: 500 ({"type"=>"illegal_state_exception", "reason"=>"Pipeline processor configured for non-existent pipeline [common]"})
> [2021-05-03T15:15:13,914][INFO ][logstash.outputs.elasticsearch] Retrying individual bulk actions that failed or were rejected by the previous bulk request. {:count=>96}

What is the output of the following from a search node?

curl -sk https://localhost:9200/_cat/indices | sort -n

so-elasticsearch-pipelines-list

**Reply by trb143** [reply]
Lots of 
 close logstash-bro-2020.03.23                   i4My4UwlRf6EpvIkEAJhpQ                              
      close logstash-bro-2020.03.24                   qQlRiO6WR8qYoauYNL27_g                              
      close logstash-bro-2020.03.25                   o04eKuhJTZG4KPEdSSEUBA                              
      close logstash-bro-2020.03.26                   5X1ZQ1gVTU-lXQ5rF5h0Zw                              
      close logstash-bro-2020.03.27                   3ZXJSP7zSbKGS4AnRiQWxA                              
      close logstash-bro-2020.03.28                   PZTz-uPHTvSYZWI3EBRxFQ                              
      close logstash-bro-2020.03.29                   u8QYnqglRUqOKae2Upsnug                              
      close logstash-bro-2020.03.30                   GMFEPkLmS2eyMXCY0j0JyQ                              
      close logstash-bro-2020.03.31                   NXpm_dYASnSmdUPvehlPNg

and 
      close logstash-ids-2020.03.23                   f6QZUjHBRhKSehWpmWf8UQ                              
      close logstash-ids-2020.03.24                   vTGVP7oOQYK2qPd3XIjlWg                              
      close logstash-ids-2020.03.25                   sT1qUienRW23BiBjgCzA3g                              
      close logstash-ids-2020.03.26                   TJhKdGj5RU-rrU1jiSXrBg                              
      close logstash-ids-2020.03.27                   QlHiSKgVQ0SR8-OdyXmuiA                              
      close logstash-ids-2020.03.28                   ue92m12vQhKU_iA3togg2g                              
      close logstash-ids-2020.03.29                   -tBa2KeZSTS18oJPSiI4Bw                              
      close logstash-ids-2020.03.30                   Wyf6GZ9UQTyX8HDDhXxOPQ                              
      close logstash-ids-2020.03.31                   flOGfbuTQcWy3nPT9b9EIg                              
      close logstash-ids-2020.04.01                   IecdWCH9Q2ayhUzV5NjiLQ                              
      close logstash-ids-2020.04.02                   85sBG5JXQkiOI5BxD1nVeg                              
      close logstash-ids-2020.04.03                   WwEd2tDCRFWTxvyE2FGozA                              
      close logstash-ids-2020.04.04                   L2aIml_YRYaB16a5b2Li0A                              
      close logstash-ids-2020.04.05                   i4DJTP3VSwuPKECvfZFgcA

as well as ossec and syslog variations.

Then we get 

      close logstash-syslog-2020.11.06                kFABey2XQyCURx6k2n12Xg                              
green open  .apm-agent-configuration                  uwahtNAYTNyBV5zRVoHi1g 1 0    0    0    208b    208b
green open  .apm-custom-link                          XQ1bg2SaSrqS2-I6HoWiPA 1 0    0    0    208b    208b
green open  .async-search                             l3MCMRmVTbOBUKXJwU5Wog 1 0    0    0   3.4kb   3.4kb
green open  elastalert_status_error                   e0czrIAnSX-E-JfUQPO7cg 1 0    0    0    208b    208b
green open  elastalert_status_past                    pzxOFVXqRn-3-i0EY5qgCA 1 0    0    0    208b    208b
green open  elastalert_status_silence                 76BDeL3nSAuV3XsikW9uag 1 0    0    0    208b    208b
green open  elastalert_status_status                  oqQqbSEjSgWnVzxIS3nFmg 1 0    0    0    208b    208b
green open  elastalert_status                         ViWLRjkYSH6nE_zCFua5HQ 1 0    0    0    208b    208b
green open  .kibana_1                                 sSJeuytPRnOr9X5cMFtseQ 1 0  539    0 492.7kb 492.7kb
green open  .kibana_2                                 4nsSJgTBQcuWJpaPP4McSw 1 0  550    1 615.4kb 615.4kb
green open  .kibana_3                                 uHV68M0zRyG0nNQTz8kA0g 1 0  564    1 626.7kb 626.7kb
green open  .kibana_4                                 EhXCRJ8eR1SStcfwNDWBoQ 1 0  572  125 977.1kb 977.1kb
green open  .kibana_5                                 o3ZrI8_jSQmup5GcesoDow 1 0 1585   94 706.9kb 706.9kb
green open  .kibana_6                                 5zyG8cEpQxW1T2FSIBnh-Q 1 0 3167    0   2.7mb   2.7mb
green open  .kibana-event-log-7.11.2-000001           CBhruuOCStmkgxoO88_Ozw 1 0    2    0    11kb    11kb
green open  .kibana_task_manager_1                    1FSGt5miQn2CUAywwG5eTg 1 0    0    0    208b    208b
green open  .kibana_task_manager_2                    lP-q5Z3iQEG9V1iejwTUNA 1 0    9 1299   1.3mb   1.3mb
green open  metrics-endpoint.metadata_current_default T6UPQVLzQtGwztHmdayDLw 1 0    0    0    208b    208b
green open  .security-7                               I3rVxO7yR66FRJmdJiJ_9Q 1 0   48    0 164.5kb 164.5kb
green open  .tasks                                    NlY1NgotSKukIAO4fxtUmw 1 0    2    0  13.7kb  13.7kb
green open  .transform-internal-005                   8DHLM6y_QYOSf0Vtc2lNGg 1 0    3    0  23.7kb  23.7kb

I have run so-elastic-clear 

so-pipelines-lis gives

merlin@trinity-ids:~$ so-elasticsearch-pipelines-list
[
  "common.nids",
  "dns.tld",
  "filterlog",
  "http.status",
  "import.wel",
  "logs-endpoint.alerts-0.17.1",
  "logs-endpoint.diagnostic.collection-0.17.1",
  "logs-endpoint.events.file-0.17.1",
  "logs-endpoint.events.library-0.17.1",
  "logs-endpoint.events.network-0.17.1",
  "logs-endpoint.events.process-0.17.1",
  "logs-endpoint.events.registry-0.17.1",
  "logs-endpoint.events.security-0.17.1",
  "logs-system.application-0.10.9",
  "logs-system.auth-0.10.9",
  "logs-system.security-0.10.9",
  "logs-system.syslog-0.10.9",
  "logs-system.system-0.10.9",
  "metrics-endpoint.metadata-0.17.1",
  "metrics-endpoint.metrics-0.17.1",
  "metrics-endpoint.policy-0.17.1",
  "osquery.live_query",
  "osquery.normalize",
  "osquery.query_result",
  "ossec",
  "strelka.file",
  "suricata.alert",
  "suricata.common",
  "suricata.dhcp",
  "suricata.dnp3",
  "suricata.dns",
  "suricata.fileinfo",
  "suricata.flow",
  "suricata.ftp",
  "suricata.ftp_data",
  "suricata.http",
  "suricata.ikev2",
  "suricata.krb5",
  "suricata.nfs",
  "suricata.rdp",
  "suricata.sip",
  "suricata.smb",
  "suricata.smtp",
  "suricata.snmp",
  "suricata.ssh",
  "suricata.tftp",
  "suricata.tls",
  "syslog",
  "sysmon",
  "win.eventlogs",
  "xpack_monitoring_6",
  "xpack_monitoring_7",
  "zeek.common",
  "zeek.common_ssl",
  "zeek.conn",
  "zeek.dce_rpc",
  "zeek.dhcp",
  "zeek.dnp3",
  "zeek.dns",
  "zeek.dpd",
  "zeek.files",
  "zeek.ftp",
  "zeek.http",
  "zeek.intel",
  "zeek.irc",
  "zeek.kerberos",
  "zeek.modbus",
  "zeek.mysql",
  "zeek.notice",
  "zeek.ntlm",
  "zeek.pe",
  "zeek.radius",
  "zeek.rdp",
  "zeek.rfb",
  "zeek.signatures",
  "zeek.sip",
  "zeek.smb_files",
  "zeek.smb_mapping",
  "zeek.smtp",
  "zeek.snmp",
  "zeek.socks",
  "zeek.software",
  "zeek.ssh",
  "zeek.ssl",
  "zeek.syslog",
  "zeek.tunnel",
  "zeek.tunnels",
  "zeek.weird",
  "zeek.x509"
]

I did select "suricata instead of zeek for metadata.

**Reply by trb143** [reply]
we only have a standalone server as we are a small organization.  The box has 32GB ram.
